app.py

```diff
@@ -44,11 +44,32 @@ def experience():
     '''
     Handle experience requests
     '''
-    if request.method == 'GET':
-        return jsonify()
 
-    if request.method == 'POST':
-        return jsonify({})
+    if request.method == "GET":
+        experiences: list[Experience] = data.get("experience", [])
+        if index := request.args.get("index") is not None:
```

Comment:
Do the listed issues talk about this update? I think this might be relevant for some other issue.